wadas/domain/animal_detection_mode.py

```diff
@@ -16,6 +16,11 @@ def __init__(self, classification=True):
         super(AnimalDetectionAndClassificationMode, self).__init__()
         self.process_queue = True
         self.en_classification = classification
+        self.type = (
+            OperationMode.OperationModeTypes.AnimalDetectionAndClassificationMode
+            if classification
+            else OperationMode.OperationModeTypes.AnimalDetectionMode
+        )
```

Comment:
you can rewrite by not using a ternary operation but a normal if/else clause that won't overflow the max line